# HG changeset patch
# User Ray Donnelly <mingw.android@gmail.com>
# Date 1471185125 -3600
#      Sun Aug 14 15:32:05 2016 +0100
# Node ID 752045fad267219219275ae62ca4af4d1673978b
# Parent  aed778a98a9b66ab7759a978eb2d930fed8c6c84
Win32: Add missing #include <stdlib.h>

This is needed for the declaration of malloc(). On other systems it
is included somewhere in the hierarchy, but not under Win32/mingw-w64

diff --git a/rpy/rinterface/r_utils.c b/rpy/rinterface/r_utils.c
--- a/rpy/rinterface/r_utils.c
+++ b/rpy/rinterface/r_utils.c
@@ -23,6 +23,7 @@
 #include <Rdefines.h>
 #include <R_ext/Rallocators.h>
 #include "r_utils.h"
+#include <stdlib.h>
 
 static int embeddedR_isinitialized;
 
